### DISCUSSION TITLE
The question about a large number of OUTCAR when using dpdata

### DISCUSSION THREAD

**Discussion by KangBaoBBS** [deepmodeling/dpdata#666] [Q&A]
If I have lots of OUTCAR, can I merge datasets together at the time of dpdata-conversion?
For example, there are 3000 OUTCAR. All the OUTCAR contains 1 frame. When I use dpdata for 3000 OUTCAR, it will be 3000 "deepme_data" folder paths. It is difficult to set them in "input.json" and separate them into "training_data" and "validation_data".
What can I do for the large numbers OUTCAR? Can I combine OUTCARs with the same molecular formula together? I means that the same "type.raw" and "type_map.raw". Or can I use the command "cat OUTCAR1 OUTCAR2 > OUTCAR" to combine OUTCAR with same "type.raw" and "type_map.raw"?
Thank you and Looking forward to your reply.

**Comment by wanghan-iapcm** [comment]
you may append a dpdata `LabeledSystem` to an existing `LabeledSystem`. 
see https://docs.deepmodeling.com/projects/dpdata/en/master/api/dpdata.html#dpdata.LabeledSystem

**Reply by KangBaoBBS** [reply]
Thank you! I will try it.

**Comment by njzjz** [comment]
Btw, DeePMD-kit can process 3000 systems. I once used it to train against 48055 systems.
